ask-sdk-dynamodb-persistence-adapter/src/com/amazon/ask/attributes/persistence/impl/utils/ItemUtilsV2.java

```diff
@@ -0,0 +1,220 @@
+// CHECKSTYLE:OFF
+/*
+    Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file
+    except in compliance with the License. A copy of the License is located at
+
+        http://aws.amazon.com/apache2.0/
+
+    or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+    the specific language governing permissions and limitations under the License.
+ */
+
+package com.amazon.ask.attributes.persistence.impl.utils;
+
+import java.math.BigDecimal;
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import software.amazon.awssdk.core.SdkBytes;
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+
+/**
+ * A port of the AWS SDK for Java v1 ItemUtils class which is currently
+ * unavailable in the v2 SDK and is necessary to convert between higher level
+ * DynamoDB wrapped DAO types and generic Java types.
+ *
+ * Once this class is available in the v2 AWS SDK this implementation should be
```

Comment:
This class seems very similar to the itemUtil in V1 aws-sdk except it import the base type from v2 aws sdk. Just have one question: is it means developer who use this class would need to update their code to import it from aws-sdk in furture ?

Reply:
By `this class`, you mean `ItemUtils` from v1 sdk? or `ItemUtilsV2` from our SDK ? If it's the former, I don't think AWS SDK v2 will have that util class added in the SDK in the near future. In case they will eventually add it, I think all we need to do is do the right import (considering the function signatures remain the same) and it should be seamless enough.

Reply:
Ahh, I was refering to the ItemUtilsV2 from our SDK. As Jonathan mentioned he will remove ItemUtilsV2 from our SDK once aws SDK provide it. Thus people who import the those utils from our pkg would need to update their code to import from aws sdk right ?  But seems aws sdk doesn't have plan to provide it lol